assets/src/application/ui/display/DebugInfo.tsx

```diff
@@ -41,7 +41,7 @@ const DebugInfo: React.FC<DebugInfoProps> = ({ data, asJson = true, asCollapse =
 
 	return (
 		<>
-			<Button buttonText={buttonText} onClick={handleToggle} size='md' mt='2em' />
+			<Button className='ee-debug-info-btn' buttonText={buttonText} onClick={handleToggle} size='md' mt='2em' />
```

Comment:
good observation, removed these altogether because font-size is used from the generic btn styles and because margin top doesn't feel quite right, instead added a bit of padding to the div above the button - `.sections-wrapper`